### PR TITLE
OF-1816: Close input streams in log.jsp correctly

### PR DESCRIPTION
**Pull request by cpetzka**
We had the problem that the logfiles sometimes cannot be loaded correctly. This pull request solves this problem.

### CODE REVIEW

**Comment by guusdk** [comment]
Ah, this seems like a sensible improvement. Can you tell me exactly what happened, as I never had the problem before?

Also, I suggest that you implement Java's try-with-resources, to do the same thing (ensure that streams are closed), but with a lot less code (which is easier to maintain).

**Comment by guusdk** [comment]
I've created a new issue for this problem in our bugtracker: https://issues.igniterealtime.org/browse/OF-1816

**Comment by cpetzka** [comment]
Sometimes we noticed that the log content in the admin console was empty even though the log files were not empty. Only restarts helped to display the log content again. After these changes were applied, the problem did not occur again.

I will adapt the code to your recommendations.

File: xmppserver/src/main/webapp/log.jsp [hunk 1]
```diff
@@ -109,36 +109,17 @@
...
         String line;
         int totalNumLines = 0;
-        BufferedReader in = null;
-        InputStreamReader inputStreamReader= null;
-        FileInputStream fileInputStream=null;
-        try{
-        	fileInputStream = new FileInputStream(logFile);
-        	inputStreamReader = new InputStreamReader(fileInputStream, "UTF-8");
-        	in = new BufferedReader(inputStreamReader);
+        try(FileInputStream fileInputStream = new FileInputStream(logFile); InputStreamReader inputStreamReader = new InputStreamReader(fileInputStream, "UTF-8"); BufferedReader in = new BufferedReader(inputStreamReader);){
```
Comment (guusdk) [inline xmppserver/src/main/webapp/log.jsp]: Ugh. Mind adding some newlines? Something like this is perfectly valid.
```
try( FileInputStream fileInputStream = new FileInputStream(logFile);
     InputStreamReader inputStreamReader = new InputStreamReader(fileInputStream, "UTF-8");
     BufferedReader in = new BufferedReader(inputStreamReader); )
{
```

Comment (cpetzka) [reply xmppserver/src/main/webapp/log.jsp]: My mistake! I disabled automatic code formatting to keep whitespace changes at a minimum.

Comment (cpetzka) [reply xmppserver/src/main/webapp/log.jsp]: done.
Should I make a new pull request with only one commit?

Comment (guusdk) [reply xmppserver/src/main/webapp/log.jsp]: Don't bother, this is fine.

**Comment by guusdk** [comment]
Thanks! I've cherry-picked this change to the 4.4 branch too.

**Pull request merged**